backend/app/views/spree/admin/return_authorizations/_form.html.erb

```diff
@@ -37,7 +37,7 @@
           </td>
           <td><%= inventory_unit.state.humanize %></td>
           <td class="charged-amount" data-charged-amount="<%= return_item.pre_tax_amount %>">
-            <%= return_item.display_pre_tax_amount %>
```

Comment:
@PSzyszkaSpark we need to keep the logic to display item price before taxation here